Library/Formula/sent.rb

```diff
@@ -0,0 +1,14 @@
+class Sent < Formula
+  desc "Simple plaintext presentation tool."
+  homepage "http://tools.suckless.org/sent/"
+  url "http://dl.suckless.org/tools/sent-0.2.tar.gz"
+  sha256 "53b961f9d92a277a6408df7025b4a6deae6b655a797383c93442290e45391076"
+
+  depends_on :x11 => :build
+
+  def install
+    system "make"
+    bin.install "sent"
```

Comment:
I looked for it and it didn't seem to exist. Now I looked again and there is indeed an install rule in the `Makefile`. I will adapt it to work with that, it's cleaner this way.